features/step_definitions/logging.rb

```diff
@@ -705,3 +705,37 @@
     end
   end
 end
+
+# Delete the fluentd buffer files in nodes. the fluentd must be deleted before running this step.
+Given /^logging collector bufferfiles are cleared on all nodes$/ do
+  ensure_admin_tagged
+  serviceaccount="collector-clean"
+  ds_name="collector-clean"
+  user(0)
+  step %Q/I have a project/
+
+  fluentd_pods = BushSlicer::Pod.get_labeled("logging-infra=fluentd", project: project("openshift-logging", switch: false), user: admin)
+  if(fluentd_pods.length()>0)
+    puts("The fluetnd pods is still running")
+    @result[":success"] = false
```

Comment:
I think here can use `raise "The fluentd pods is still running"`

Reply:
i don't want to exit the test, so I don't use raise here.

Reply:
Then if the fluentd pods are running, the step won't do anything.

Reply:
What's the usage of `@result[":success"] = false` ?

Reply:
@result[":success"] should be bool value.   If we set false here.  We can use the following step to check the status as usual.

Given /^the step (succeeded|failed)$/ do |outcome|
  if (outcome == "succeeded") != @result[:success]
    raise "the step #{@result[:success] ? "succeeded" : "failed"}"
  end
end

    raise "the step #{@result[:success] ? "succeeded" : "failed"}"
  end
end